package.json

```diff
@@ -2,7 +2,10 @@
   "name": "svelte-jester",
   "version": "1.7.0",
   "description": "A Jest transformer for Svelte - compile your components before importing them into tests",
-  "main": "src/transformer.cjs",
+  "main": "src/transformer.js",
```

Comment:
```suggestion
  "main": "src/transformer.js",
  "type": "module",
```

Reply:
`type: module` should be there already https://github.com/mihar-22/svelte-jester/blob/master/package.json#L6